test/runtests.jl

```diff
@@ -35,44 +40,57 @@ end
     cd(dirname(@__DIR__)) do
         datadir = joinpath("test", "data")
         # Process a saved set of coverage data...
-        r = process_file(joinpath(datadir,"Coverage.jl"))
+        r = process_file(joinpath(datadir, "Coverage.jl"))
         # ... and memory data
         analyze_malloc(datadir)
         lcov = IOBuffer()
         # we only have a single file, but we want to test on the Vector of file results
         LCOV.write(lcov, FileCoverage[r])
-        expected = read(joinpath(datadir, "expected.info"), String)
+        expected = read(joinpath(datadir, "tracefiles", "expected.info"), String)
         if Sys.iswindows()
             expected = replace(expected, "SF:test/data/Coverage.jl\n" => "SF:test\\data\\Coverage.jl\n")
         end
         @test String(take!(lcov)) == expected
+        # test that reading the LCOV file gives the same data
+        lcov = LCOV.readfolder(datadir)
```

Comment:
So... how is one supposed to use `readfolder` later on? Call it in addition (or instead of) `process_folder` from the CI scripts (but then, why isn't `etc/travis-coverage.jl` modified to do so)? Or is this code simply not yet done, and hence not hooked up in `process_folder`? Or what?

Reply:
In the readme, I show using both now (in tandem), with a few comments mentioning the applicability of each line. The CI script doesn't need to change, since, for most users, this addition is 6 of one or half a dozen of the other. It really only affects people who are using it locally (don't have a CI sandbox environment) or are measuring Base and stdlib coverage.

Reply:
OK